ci-operator/config/openshift/knative-client/openshift-knative-client-release-v0.2.0.yaml

```diff
@@ -0,0 +1,17 @@
+tag_specification:
```

Comment:
This config builds no images, is that intentional?

Reply:
yes, it doesnt build any images, its a CLI.